thetagpu/README.md

```diff
@@ -22,23 +22,23 @@ With this environment loaded, users will need to build and install both SmartSim
 SmartRedis through pip. Users are advised to build on
 compute nodes, as the service nodes do not have all needed libraries installed.
 Usually we recommend users installing or loading miniconda and
-using the pip that comes with that installation. 
+using the pip that comes with that installation.
 
 The following commands were utilized to build SmartSim on Theta,
 after the correct conda environment was loaded.
 
 ```bash
 export CC=$(which gcc)
 export CXX=$(which g++)
-conda install swig cmake git-lfs 
+conda install swig cmake git-lfs
 conda install pytorch==1.7.1 torchvision==0.8.2 torchaudio==0.7.2 cudatoolkit=11.0 -c pytorch -y
-pip install . tensorflow==2.4.2 numpy==1.19.5 onnx==1.7 
+pip install . tensorflow==2.4.2 numpy==1.19.5 onnx==1.7
```

Comment:
Here too, I think we can just use `pip install .[ml]`

Reply:
Addressed.